e2e/cypress/integration/search/results_post_comment_spec.js

```diff
@@ -11,11 +11,18 @@
 // Group: @search
 
 describe('Search', () => {
-    it('S14548 Search results Right-Hand-Side: Post a comment', () => {
-        // # Login and navigate to the app
-        cy.apiLogin('user-1');
-        cy.visit('/ad-1/channels/town-square');
+    before(() => {
+        // # Login as test user and visit town-square
+        cy.apiInitSetup({loginAfter: true}).then(({team}) => {
+            cy.visit(`/${team.name}/channels/town-square`);
 
+            Cypress._.times(5, () => {
+                cy.postMessage(`asparagus${Date.now()}`);
```

Comment:
This is being done during the test a few lines down, how come it needs to be done twice?

Reply:
During test verification, I realized that there's only the test message posted in a team. So, I intentionally added those to add complexity when searching for a post. I should have added a comment but now updated with:"Post several messages of similar format to add complexity in searching".
Also, I changed to `getRandomId()` since `Date.now()` did not generate different values in a loop.